docs/user/publishing-feeds.md

```diff
@@ -1,16 +1,37 @@
 # Publishing Feeds
 
-## Publishing a Feed Version
+## Publishing a Feed Version Manually
+
 Once a feed version has been loaded into the manager either through direct upload, URL fetch or loaded from the editor, in order to make this feed active (i.e., available for creating alerts or visible in other 511 systems) you must first publish the feed.  This step allows users to review the feed contents (such as number of routes or stops) as well as any validation issues before beginning the publication process.
 
 Once you have checked these details, you must click the yellow `Publish to MTC` button.
 
 ![publish feed version](../img/feed-manager-publish.png)
 
+## Auto Publishing Feed Versions Fetched Automatically
+
+To avoid repeating the manual publishing steps above, 
+you can set up auto publishing for feeds that are to be fetched automatically.
+
+When creating new feeds, the auto-publishing setting is under the `Automatic publishing` section.
+For existing feeds, the auto-publishing setting is found under the `Settings` tab, `Auto-publish` category.
+
+![Auto-publish settings](../img/auto-publish-feed.png)
+
+In the list of feed sources, feeds that are set to be auto-published have an indicator displayed the feed name.
```

Comment:
Think the word "below" might have been missed.
```
In the list of feed sources, feeds that are set to be auto-published have an indicator displayed below the feed name.
```

Reply:
Good catch! Thank you, fixed in https://github.com/ibi-group/datatools-ui/pull/751/commits/add3f0bc629feb9e14309b728be9311266e56f53.